content/docs/command-reference/exp/init.md

```diff
@@ -18,11 +18,11 @@ usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
 
 ## Description
 
-`dvc exp init` helps you get started with DVC Experiments quickly. It reduces
-boilerplate DVC procedures by creating a `dvc.yaml` file that assumes standard
-locations of your input data, <abbr>parameters</abbr>, source code, models,
-<abbr>metrics</abbr> and [plots](/doc/command-reference/plots). These locations
-can be customized through the [options](#options) below or via
+`dvc exp init` helps you get started with DVC Experiments quickly. It creates an
+experiment pipeline in a dvc.yaml file, which assumes standard locations for the
```

Comment:
@dberenbaum I don't think we should use the pipeline concept here, WDYT?
```suggestion
`dvc exp init` helps you get started with DVC Experiments quickly. It creates an
experiment inside a `dvc.yaml` file, which assumes standard locations for the
```

Reply:
Also need backticks around `dvc.yaml` whatever we decide.

Reply:
> dvc exp init doesn't create an experiment

It "creates" it in the sense that it codifies it but yeah since it's not run yet maybe that wasn't the best wording... 🤔

Reply:
Committing this for now:

```suggestion
`dvc exp init` helps you get started with DVC Experiments quickly. It prepares an
experiment in a `dvc.yaml` file, which assumes standard locations for the
```